### ISSUE TITLE
Images miss alt text, are not visible in most email clients

### ISSUE THREAD

**Issue by weeblr** [freescout-helpdesk/freescout#2265]
Hi

We have that issue with Freescount where we often link images in tickets replies, using the standard editor:

![image](https://user-images.githubusercontent.com/5846185/191564787-382eaf70-5604-42d1-a6e4-404102abd9de.png)

That editor does not allow entering an alt text for the image and this causes 2 issues:

1. It's not accessible when read by the customer
2. Importantly, as most email client hide images by default, customer does not even know there's an image in our reply. Here is an example:

![image](https://user-images.githubusercontent.com/5846185/191565135-1256f70f-92ac-4d07-8435-7e8a2553af42.png)

After enabling image display, the message looks like this:

![image](https://user-images.githubusercontent.com/5846185/191565316-e01e98d1-612f-4e2a-ac36-ca264333b2cc.png)

This is quite an issue, as it may change the message meaning completely.

Here is how it looks when an alt text is provided with an image and images are not displayed:

![image](https://user-images.githubusercontent.com/5846185/191565929-0e2f479e-67cc-42ad-9daa-0c7b51f92cc1.png)

Hope you can address this (or maybe I missed something?).

Cheers

**Comment by weeblr** [comment]
Hi @freescout-helpdesk 

Thanks for that quick patch, but I just had a look at the code and I don't know how what you did is related to the issue I raised?

Your patch adds an automated alt text - based on the file name - in the text editor.

This is not what this issue is about. This issue is about how email clients render messages sent by Freescout. So:

1 - In any case, it's in the email content that an alt text should be added, although it's of course also a good thing to that in the editor and when the messages are displayed on screen inside of Freescount

2 - The file name is usually not a good alt text. In our case, they are mostly screenshot with names such as `/images/screenshots/company-name-2022-09-23-09.33.50.png`. Using the filename is better than nothing but it really should be a input field in the image insertion dialog.

I can see #2 is more difficult to implement but at least consider fixing the actual issue by injecting the alt tag in email messages.

Thanks and regards,

**Comment by freescout-helpdesk** [comment]
Feel free to submit a feature request: https://freescout.net/request-feature/

**Comment by weeblr** [comment]
Hi

Hmm, looking at that list, it does not seem useful to submit a feature request. 

In addition, accessibilty is not really a feature request. Some of my clients are, today, getting lawyers letter threatening to sue because their website have accessibility issues. 

I have bought most of your modules and planned on using the end user portal. But I realize now that I have not tested it for accessibility and I think I have to do that now, as it's going to be public-facing. 

Do you do accessibility testing?

Best regards
